### ISSUE TITLE
API Auth docs

### ISSUE THREAD

**Issue by av** [ai-dock/base-image#7]
Firstly, thank you again for this comprehensive solution!

Secondly, thanks for providing so many auth options, API-only auth was exactly what I needed for one of the projects

https://github.com/ai-dock/base-image/blob/b8bc35fa2f941339d0895e99d98e4f0f22593957/build/COPY_ROOT_0/opt/caddy/share/base_config#L26

And lastly, unfortunately it took quite a bit of time to find that these optionsa are available in the first place, so some wiki reference would really useful

**Comment by av** [comment]
Side note, certain services (Open WebUI) do not allow configuring additional auth params for ComfyUI, so the only way to integrate is currently by disabling the auth altogether.

Is it possible to also enable basic HTTP auth support for caddy? I.e. 
```
username:password@domain
```
I understand that it might not be desirable from the project's point of view, but it'd help in cases like above where `?token` can't be used

**Comment by robballantyne** [comment]
Yes both of these are valid issues that I'm hoping to address shortly.

Depending on your runtime environment, you might find that it's appropriate to create a SSH tunnel for the proxied upstream - there's no Auth issue then as the upstream is only ever exposed externally via caddy

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
